mailscanner/functions.php

```diff
@@ -4740,7 +4740,7 @@ function printTrafficGraph()
     );
     $graphgenerator->graphTitle = '';
     $graphgenerator->settings['timeInterval'] = 'PT' . $graphInterval . 'M';
-    if ($graphInterval < 240) {
+    if ($graphInterval < 360) {
```

Comment:
Hello @Skywalker-11  

Why do you stay with the test?

I will remove it completely.

With 
```
        $graphgenerator->settings['timeScale'] = 'PT1M';
        $graphgenerator->settings['timeGroupFormat'] = 'Y-m-dTH:i:00';
        $graphgenerator->settings['timeFormat'] = 'H:i';
```
![capture d ecran 2017-06-29 a 19 08 08](https://user-images.githubusercontent.com/12907102/27700607-6921c002-5cfe-11e7-93e1-5439b584af45.png)
![capture d ecran 2017-06-29 a 19 20 41](https://user-images.githubusercontent.com/12907102/27701092-220bf348-5d00-11e7-84b4-105c5fc253e3.png)

with 
```
        $graphgenerator->settings['timeScale'] = 'PT1H';
        $graphgenerator->settings['timeGroupFormat'] = 'Y-m-dTH:00:00';
        $graphgenerator->settings['timeFormat'] = 'H:00';
```
![capture d ecran 2017-06-29 a 19 07 34](https://user-images.githubusercontent.com/12907102/27700606-691cf5fe-5cfe-11e7-951f-84624c35eddf.png)

In fact, it will be nice to have the first graph with the time legend of the second.  😎